### ISSUE TITLE
Planned support for OpenAPI 3.1?

### ISSUE THREAD

**Issue by bertrand-caron** [Mermade/oas-kit#529]
The OpenAPI specification 3.1 was released on February the 18th, 2021. 

Are there any plans for this library to support it?

Thanks :)

**Comment by crutch12** [comment]
@MikeRalphson what do you think?

**Comment by MikeRalphson** [comment]
Yes, support is planned, and under way, but there is one major outstanding issue, getting `$ref`s to work correctly in the new JSON Schema compatible way.

**Comment by JaredAAT** [comment]
Hey @MikeRalphson, long time fan of this library.  Wonder if you have an update as to where openAPI 3.1.0 support is.  Is there a branch/repo that I could look at?

**Comment by MikeRalphson** [comment]
I'll see if I can push out an interim release with OAS 3.1 support as-is.

**Comment by meden** [comment]
Hello, any news on this? :slightly_smiling_face:

**Comment by tonyxiao** [comment]
is there an oas 3.0 to oas 3.1 converter we can use for this?

**Comment by chrisolson443** [comment]
+1 for needing 3.1 compatibility! 🙏

**Comment by matt-duffett** [comment]
+1 for 3.1 compatibility!
